src/server.ts

```diff
@@ -31,12 +33,67 @@ const { handleCustomResponse } = require('./custom_response');
 const DEFAULT_AUTH_REALM = 'ProxyChain';
 const DEFAULT_PROXY_SERVER_PORT = 8000;
 
+type ConnectionStats = {
+    srcTxBytes: number;
+    srcRxBytes: number;
+    trgTxBytes: number | null;
```

Comment:
maybe we should mark the nullable properties as optional too?

Reply:
No, it's either number or null. It's never undefined. I was thinking about moving to undefined but that would be breaking.